Makefile

```diff
@@ -0,0 +1,36 @@
+.DEFAULT_GOAL := all
+CHART := ccd-user-profile-importer
+RELEASE := chart-${CHART}-release
+NAMESPACE := chart-tests
+TEST := ${RELEASE}-test-service
```

Comment:
There are no tests ? May be move the tests from where this is copied? hoping it has some